README.md

```diff
@@ -62,6 +62,73 @@ This starts a preview server on your machine, and watches all files for changes.
 
 The preview server supports hot reloading. Once the server is running on your machine, any changes you make to Markdown content will appear automatically in your browser within a few seconds. Note that changes to page frontmatter or site configuration require you to stop and start the preview server.
 
+## Windows 
```

Comment:
I suggest adding a `Troubleshooting` section, and put this content there. If anything else shows up (regardless of the env) in the future can be added into that section.

Reply:
Retty had the same Idea, I'll update.